.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -16,7 +16,7 @@ env:
   TARGET_ORG: actions-runner-controller
   TARGET_REPO: arc_e2e_test_dummy
   IMAGE_NAME: "arc-test-image"
-  IMAGE_VERSION: "dev"
+  IMAGE_VERSION: "0.4.0"
```

Comment:
Why are we pinning the image version here?

Reply:
We need to do this because the build.Version will be replaced with "dev"
Chart.yaml uses "0.4.0" while build.Version uses dev, causing mismatch each time. So no e2e test can run because whenever we create something, we delete it on the first pass in the controller

Reply:
Clear, how are we going to maintain this down the line or it's inconsequential because we're not really pulling anything and we're building the image locally?

Reply:
Each time we bump the chart version, e2e IMAGE_VERSION must align with it. That is it :relaxed: